geometry_msgs/PoseStamped start
geometry_msgs/PoseStamped goal
---
geometry_msgs/Pose[] pose_list
bool normal_goal
